processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -399,6 +416,35 @@ public String apply(@Nullable String input)
     return makeIndexFiles(indexes, outDir, progress, mergedDimensions, mergedMetrics, rowMergerFn, indexSpec);
   }
 
+  public static File reprocess(final File inDir, final File outDir, final IndexSpec indexSpec) throws IOException
+  {
+    return reprocess(inDir, outDir, new BaseProgressIndicator(), indexSpec);
+  }
+
+  public static File reprocess(
```

Comment:
I had a justifyable reason to put it there when I stuck it there, but cannot remember what it is now. Will move